openassessment/xblock/static/css/openassessment-ltr.css

```diff
@@ -1,3 +1,3 @@
ata:last-child,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata:last-child,.step--student-training .student-training__assessment .question__answers .wrapper--metadata:last-child{margin-right:0}}@media screen and (min-width: 925px) and (max-width: 1299px){.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata,.step--student-training .student-training__assessment .question__answers .wrapper--metadata{float:left;display:block;margin-right:2.35765%;width:65.88078%;margin-bottom:0}.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata:last-child,.step--student-training .student-training__assessment .question__answers .wrapper--metadata:last-child{margin-right:0}}@media screen and (min-width: 1300px){.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata,.step--student-training .student-training__assessment .question__answers .wrapper--metadata{float:left;display:block;margin-right:2.35765%;width:65.88078%;margin-bottom:0}.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata:last-child,.step--student-training .student-training__assessment .question__answers .wrapper--metadata:last-child{margin-right:0}}.openassessment .staff-assessment__assessment .question__answers .answer__tip,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .answer__tip,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .answer__tip,.step--student-training .student-training__assessment .question__answers .answer__tip,.openassessment .staff-assessment__assessment .question__answers .answer__points,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .answer__points,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .answer__points,.step--student-training .student-training__assessment .question__answers .answer__points{display:block}.openassessment .staff-assessment__assessment .question__answers .
```

Comment:
I'm surprised there is built CSS and JS checked into git here. Is there a reason for that? It's usually an anti-pattern because of these giant impossible to read diffs that are created on every little sass/js change.